package.json

```
@@ -132,7 +132,7 @@
   },
   "engines-next": {
     "description": "Versions other than the below are deprecated and a warning will be logged",
-    "node": "^16.13.0 || >= 18.12.0"
+    "node": ">= 18.12.0"
```

```suggestion
    "node": ">= 18.12.0"
```

Time to revisit the question, should this instead be `^18.12.0`?

---

sounds good, only support lts and warn otherwise